pages/sync/tokens-studio.en.mdx

```diff
@@ -0,0 +1,57 @@
+# Tokens Studio Platform sync setup guide
+Tokens Studio has a standalone web-based platform for dynamic creation and management of design decisions, currently in Beta! 
+
+→ More information on the Studio Platform and sign up for Beta access here. TODO
```

Comment:
@six7 reminder to add the URL hyperlink where it says `here` and remove the TODO in the doc.

Reply:
```suggestion
→ More information on the Studio Platform and sign up for Beta access [here](https://tokens.studio/studio).
```